BUILD.gn

```diff
@@ -156,18 +159,49 @@ asar("js2asar") {
       "lib/browser/api/views/text-field.js",
     ]
   }
+
+  outputs = [
+    "$target_gen_electron_js/{{source}}",
+  ]
+}
+
+asar("electron.asar") {
+  deps = [
+    ":copy_lib_js",
+  ]
+
+  root = "$target_gen_electron_js/electron/lib"
+  sources = get_target_outputs(deps[0])
   outputs = [
     "$root_out_dir/resources/electron.asar",
   ]
-  root = "lib"
 }
 
-asar("app2asar") {
+copy("copy_default_app_js") {
   sources = filenames.default_app_sources
+  outputs = [
+    "$target_gen_default_app_js/{{source}}",
+  ]
+}
+
+copy("copy_default_app_octicon_deps") {
+  sources = filenames.default_app_octicon_sources
+  outputs = [
+    "$target_gen_default_app_js/electron/default_app/octicon/{{source_file_part}}",
```

Comment:
Instead of some magical `node_modules` redirect / copy here we put it in a reasonable folder "octicon" and load relative to `__dirname` in code